backend/app/controllers/api/v1/participant/studies_controller.rb

```diff
@@ -37,19 +37,19 @@ def launch
   end
 
   def land
-    if params[:metadata]
+    if params[:md]
       ParticipantMetadatum.create!(
         user_id: current_user.id,
         study_id: @study.id,
-        metadata: params[:metadata]
+        metadata: params[:md]
       )
     end
     if params[:aborted]
```

Comment:
I think we'll need to check if `params[:aborted] == 'true'` here instead of just if it's present, because it could be 'false'

Reply:
yeah that's fair, but we should probably check that it's not `false` at least, that way we don't mark it as aborted when `abort=false`